### DISCUSSION TITLE
Gridviz version 3 released!

### DISCUSSION THREAD

**Discussion by joewdavies** [eurostat/gridviz#133] [Announcements]
Changelog here: https://github.com/eurostat/gridviz/blob/master/docs/changelog.md

**Comment by SPierceMyers** [comment]
Excellent work on this. I really like the concept and aesthetic.

I'm going to attempt to develop an experimental cartography application with gridviz.

Looking at the examples, my first question is about the layer re-loading between zoom levels when the grid disappears and reappears. Has this been solved by anyone? Is it an active area of development?

**Reply by jgaffuri** [reply]
Hi,
That is something we should definitively work on. Our experience is that when the map data loads quickly, it is not such a big issue. But when it takes more time, the map becomes empty when zooming in and stays empty too long. Reusing openlayer to handle that could be an option, but I am not sure if and how easy it would be. Something to keep in mind for the next releases !

**Reply by SPierceMyers** [reply]
Amazing! I am not a deep dev. I'm not capable of that, but perhaps I will be able to gain insight on this in collaboration with others who are more capable than me. As an outsider, this seems like one major issue for UX that might slow down an otherwise large wave of adoption. I think this project is really cool.
